cs.html

```diff
@@ -75,468 +81,147 @@ <h2>Terminology</h2>
-    <section id="quictransportbase-procedures*">
-    <h3>Procedures</h3>
-      <section>
-        <h4 id="add-bidirectional-stream-to-transport">Add QuicBidirectionalStream
-        to the QuicTransport</h4>
-        <p>To <dfn>add the QuicBidirectionalStream</dfn> to the <code><a>QuicTransportBase</a></code>
-        run the following steps:</p>
-        <ol>
-          <li>
-            <p>Let <var>stream</var> be the newly created
-            <code><a>QuicBidirectionalStream</a></code> object.</p>
-          </li>
-          <li>
-            <p>Add <var>stream</var> to <var>transport</var>'s <a>[[\QuicTransportReadableStreams]]</a>
-            internal slot. </p>
-          </li>
-          <li>
-            <p>Add <var>stream</var> to <var>transport</var>'s <a>[[\QuicTransportWritableStreams]]</a>
-            internal slot. </p>
-          </li>
-          <li>
-            <p>Continue the following steps in the background.</p>
-          </li>
-          <li>
-            <p>Create <var>stream</var>'s associated underlying data
-            transport.</p>
-          </li>
-        </ol>
-      </section>
+    <section>
+      <h2>Methods</h2>
+      <dl data-link-for="UnidirectionalStreamsTransport" data-dfn-for="UnidirectionalStreamsTransport" class=
+      "methods">
+        <dt><dfn><code>createSendStream</code></dfn></dt>
+          <dd>
+          <p>Creates an <code><a>SendStream</a></code> object.</p>
+          <p>When <code>createSendStream</code> is called, the user agent
+          <em class="rfc2119" title="MUST">MUST</em> run the following
+          steps:</p>
+          <ol>
+            <li>
+              <p>Let <var>transport</var> be the <code><a>UnidirectionalStreamsTransport</a></code>
+              on which <code>createSendStream</code> is invoked.</p>
+            </li>
+            <li>
+              <p>If <code><var>transport</var>'s state</code> is <code>"closed"</code> or
+              <code>"failed"</code>, immediately return a new <a>rejected</a> promise with a
+              newly created <code>InvalidStateError</code> and abort these steps.</p>
+            </li>
+            <li>
+	      <!-- TODO If/when we support 0-RTT, allow resoling the
+	      stream before we are connected. -->
+              <p>If <code><var>transport</var>'s state</code> is <code>"connected"</code>,
+              immediately return a new <a>resolved</a> promise with a newly created
+              <code><a>SendStream</a></code> object,
+              <a>add the SendStream</a> to the <var>transport</var>
+              and abort these steps.</p>
+            </li>
+            <li>
+              <p>Let <var>p</var> be a new promise.</p>
+            </li>
+            <li>
+              <p>Return <var>p</var> and continue the following steps in
+              the background.</p>
+            </li>
+            <ol>
+              <li>
+                <p>When <code><var>transport</var>'s state</code> transitions to
```

Comment:
I think you'll need to merge in (or copy over) the PR we merged last week that updates this to include waiting on congestion control as well. Here and also in createBidirectionalStream.

PR that merged: https://github.com/w3c/webrtc-quic/pull/120

Reply:
Done